plugin/app/feishu/feishu.go

```diff
@@ -42,6 +42,11 @@ func NewProvider() *Provider {
 	return &p
 }
 
+// Clear token cache removes cached token.
```

Comment:
exported: comment on exported method Provider.ClearTokenCache should be of the form "ClearTokenCache ..." (revive)